src/UI/Main.java

```diff
@@ -0,0 +1,43 @@
+package UI;
+
+/***
+ * @Author	 : Shiva Dhanush
+ * This file brings up the initial startSession page.
+ ***/
+ 
+import javafx.application.Application;
+import infrastructure.validation.logger.*;
+import javafx.fxml.FXMLLoader;
+import javafx.stage.Stage;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.scene.layout.BorderPane;
+import org.json.simple.JSONObject;
+
+import infrastructure.content.*;
+
+public class Main extends Application {
+	static ILogger logger = LoggerFactory.getLoggerInstance();
+	/**
+	 *This function opens the start session page.
+	 *@param PrimaryStage This is Stage for showing fxml page.
+	 *@returns nothing
+	 */
+	@Override
+	public void start(Stage primaryStage) {
+		try {
+			/** Loading the startSession page */
+			Parent root = FXMLLoader.load(getClass().getResource("StartSession.fxml"));
+			Scene scene = new Scene(root,600,800);
+			primaryStage.setScene(scene);
```

Comment:
This part was researched by madhav and anish and came to a conclusion of this dimension. I think it is better to have it like that for now as it will mess up the buttons on changing the dimensions.